tests/unit/rpc/handlers/LedgerEntryTests.cpp

```diff
@@ -2612,3 +2614,293 @@ TEST(RPCLedgerEntrySpecTest, DeprecatedFields)
     EXPECT_EQ(warning.at("id").as_int64(), static_cast<int64_t>(rpc::WarningCode::warnRPC_DEPRECATED));
     EXPECT_NE(warning.at("message").as_string().find("Field 'ledger' is deprecated."), std::string::npos) << warning;
 }
+
```

Comment:
It would be useful to add a paragraph before each test what you are trying to achieve. It would be easier for other people who come across your code who don't know about the combinations you are ttrying to test